src/components/shared/ProfileMetainfoSummary.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import React from 'react';
+
+import {
+  formatProductAndVersion,
+  formatPlatform,
+} from '../../profile-logic/profile-metainfo';
+
+import './ProfileMetainfoSummary.css';
+
+type Props = {|
+  +meta: {
+    +product: string,
+    +misc?: string,
+    +platform?: string,
+    +oscpu?: string,
+    +toolkit?: string,
+    ...
```

Comment:
Any reason not to just ProfileMeta?

Reply:
I think this makes it obvious what we use in this component. I have the various property names right in front of my eyes too.